store/src/main/java/com/nytimes/android/external/store/base/impl/RealInternalStore.java

```diff
@@ -264,6 +308,7 @@ public void clearMemory() {
         inFlightRequests.invalidateAll();
         clearDiskIfNoOp();
         memCache.invalidateAll();
+        refreshSubject.onNext(ALL_BARCODE);
```

Comment:
got away with using a single subject but filtering on a particular Barcode that signals to clear all